src/SymbolServer.jl

```diff
@@ -89,7 +90,7 @@ function getstore(ssi::SymbolServerInstance, environment_path::AbstractString, p
 
     take!(server_is_ready)
 
-    p = open(pipeline(Cmd(`$jl_cmd --code-coverage=$(use_code_coverage==0 ? "none" : "user") --startup-file=no --compiled-modules=no --history-file=no --project=$environment_path $server_script $(ssi.store_path) $pipename`, env = env_to_use), stderr = stderr_for_client_process), read = true, write = true)
+    p = open(pipeline(Cmd(`$jl_cmd --code-coverage=$(use_code_coverage==0 ? "none" : "user") --startup-file=no --history-file=no --project=$environment_path $server_script $(ssi.store_path) $pipename`, env = env_to_use), stderr = stderr_for_client_process), read = true, write = true)
```

Comment:
Why remove the no compile mode here? I think there are two benefits to it: 1) indexing packages that aren't precompiled yet is faster, 2) we won't risk precompile race conditions (if two processes try to precompile the same package, Julia errors currently).

Reply:
Ah, I disabled this for deving, will reverse!